calico_containers/tests/st/test_multi_host.py

```diff
@@ -1,50 +1,37 @@
-from test_base import TestBase
-from time import sleep
 from sh import ErrorReturnCode, ErrorReturnCode_1
+from time import sleep
+import unittest
+
+from test_base import TestBase
 from docker_host import DockerHost
 
 
+@unittest.skip("Waiting for a fix from aa1-separate-etcd")
 class MultiHostMainline(TestBase):
     def test_multi_host(self):
         """
-        Run a mainline multi-host test. Almost identical in function to the vagrant coreOS demo.
+        Run a mainline multi-host test.
+
+        Almost identical in function to the vagrant coreOS demo.
         """
         host1 = DockerHost('host1')
         host2 = DockerHost('host2')
 
         calicoctl = "/code/dist/calicoctl %s"
-
-        host1.execute(calicoctl % "reset || true")
+        try:
```

Comment:
I swear I copied this from somewhere, but I can't find it now. I'll take it out.